settings.py

```diff
@@ -24,6 +24,17 @@ class Config:
     RESTPLUS_MASK_SWAGGER = False
     REDIS_HOST = f"redis://{os.environ.get('REDIS_HOST', '127.0.0.1:6379')}"
     TRAINING_DEVICE = get_device("TRAINING_DEVICE")
+    RS_DATABUS_HOST = os.environ.get("RS_DATABUS_HOST", "127.0.0.1")
+    RS_DATABUS_PORT = int(os.environ.get("RS_DATABUS_PORT", 61613))
+    RS_DATABUS_USERNAME = os.environ.get("RS_DATABUS_USERNAME", "admin")
+    RS_DATABUS_PASSWORD = os.environ.get("RS_DATABUS_PASSWORD", "admin")
+    RS_DATABUS_SSL = not os.environ.get("RS_DATABUS_SSL") in ["0", "no"]
```

Comment:
Hmmm that is interesting. In that case, we enable SSL for everything different than ["0", "no"]. I can live with that but I wonder if that is a good practice.

Reply:
well SSL should be enabled by default regardless - it's a good practice to have overall